src/app/content/studyGuides/reducer.ts

```diff
@@ -29,10 +28,22 @@ export const initialState: State = {
 const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
   switch (action.type) {
     case getType(locationChange): {
-      const summaryShouldBeOpen = action.payload.query[modalQueryParameterName] === modalUrlName
-        && action.payload.action === 'PUSH';
+      const hasModalQuery = action.payload.query[modalQueryParameterName] === modalUrlName;
+      const summaryShouldBeOpen = hasModalQuery
+        && (action.payload.action === 'PUSH' || action.payload.action === 'REPLACE');
```

Comment:
@TomWoodward is there a way to differentiate between page reload and the back button here? we discussed this a few weeks ago and I think you had a suggestion for keeping the modal open on reload.